jenkins/catapult/compilers/ubuntu-clang/Dockerfile

```diff
@@ -28,3 +28,8 @@ RUN wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | gpg --dearmor > /etc/
 	update-alternatives --install /usr/bin/c++ c++ /usr/bin/clang++ 100 && \
 	rm -rf /var/lib/apt/lists/* && \
 	clang --version
+
+# set default user (used by jenkins)
+ARG HOME_DIR=/home/ubuntu
+USER ubuntu
```

Comment:
you don't need group for ubuntu?

Reply:
``ubuntu`` user is built into the Ubuntu 24.04 image.  Are you referring to picking a specific group for the user?